### ISSUE TITLE
PEAR.Files.IncludingFile.BracketsNotRequired for `require dirname(__FILE__) ...`

### ISSUE THREAD

**Issue by arswright** [squizlabs/PHP_CodeSniffer#2152]
What I did: Used `dirname(__FILE__)` as part of a require statement

```php
require dirname( __FILE__ ) . '/page/error-403.php';
```

What happened: PHPCS issued a warning:

```
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 18 | WARNING | [x] "require" is a statement not a function; no
    |         |     parentheses are required
    |         |     (PEAR.Files.IncludingFile.BracketsNotRequired)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

What I expected to happen: No warnings for this line of code

**Comment by gsherwood** [comment]
I can't replicate this issue with the sample code you've provided. I'm also not sure how this warning could be generated by looking at the sniff itself. The code is pretty simple:
```php
$nextToken = $phpcsFile->findNext(Tokens::$emptyTokens, ($stackPtr + 1), null, true);
if ($tokens[$nextToken]['code'] === T_OPEN_PARENTHESIS) {
    ...
}
```
It starts at the `require` token and looks for the next non-empty token. In this case, it will be the T_STRING `dirname`. Because T_STRING is not equal to T_OPEN_PARENTHESIS, it wont generate the warning.

Are you sure this is the line of code causing that error?

**Comment by arswright** [comment]
> I can't replicate this issue with the sample code you've provided. I'm also not sure how this warning could be generated by looking at the sniff itself.

Is it possible that this issue cropped up as a result of other information in memory?

I've attempted to re-create the issue and have been unable to do so - for the record, here's a link to the file where the issue was first noted (and is now non-apparent): [protected.php](https://github.com/arswright/wordpress-plugin-debugging-benchmark/blob/master/protected.php)

Supporting information:

```
phpcs --version
PHP_CodeSniffer version 3.3.1 (stable) by Squiz (http://www.squiz.net)
```

**Comment by gsherwood** [comment]
> Is it possible that this issue cropped up as a result of other information in memory?

I can't think of any reason that would happen.

Given you can't replicate and I can't figure out how it could possibly happen, I'll close this issue. If it pops up again, please comment on it and I'll re-open. Thanks for trying to replicate it again.
